src/components/DisplayArticles.jsx

```diff
@@ -36,8 +36,9 @@ const DisplayArticles = () => {
               as={Link}
               to={`/articles/${article.id}`}
               data-cy={"article-" + article.id}
-            >
-              <Image src="https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fwww.grade.com%2Fwp-content%2Fuploads%2F2020%2F03%2Fplaceholder.png" />
+            > {article.url && (
+              <Image data-cy="image" src={article.url} />
```

Comment:
are you getting the image attribute as url from the backend?

Reply:
no we get them as image attribute that contains an url. We fixed this now